addons/project-source-button/style.css

```diff
@@ -0,0 +1,12 @@
+#view-json-btn {
+  margin: 0 0 0 0.5rem;
+  border-radius: 19px;
+  background-color: #4d97ff;
```

Comment:
The button is blue, even without any custom website colors. Is that intentional? You can just use `class="button action-button"` and the button's background color and text color with change with any other userstyles, including the Scratch website's default coloring.